Sample.js

```diff
@@ -1984,10 +1985,146 @@ var app = function (request1, response) {
       logError(err);
     });
 
+  let prevStep = printMetadata2;
+  // Community steps
+  if (communityId) {
+    // Step 18
+    var getTenantRoles = prevStep.then(
+      // getting tenant roles
+      function (res) {
+        console.log('\nGetting tenant roles');
+        if (client.tokenExpires < nowSeconds) {
+          return checkTokenExpired(client)
+            .then(function (res) {
+              refreshToken(res, client);
+              return client.getTenantRoles(tenantId);
+            })
+            .catch(function (err) {
+              logError(err);
+            });
+        } else {
+          return client.getTenantRoles(tenantId);
+        }
+      }
+    );
+
+    var communityMemberRoleTypeId = 'f79a55da-7c76-4600-a809-0f62ca9971d9';
```

Comment:
constant vars like this in most languages go at the top and designated with different naming is that true here?

Reply:
such as my favorite one, SCREAMING_SNAKE_CASE :D

Reply:
Good point, moved this to the top